addons/l10n_eg_pos_edi_eta/views/pos_config_views.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+        <record id="pos_config_view_form" model="ir.ui.view">
```

Comment:
id is a bit too generic, we could have a clash can you add l10n_eg_pos_edi_eta in it